doc/nrf/device_guides/working_with_nrf/nrf52/developing.rst

```diff
@@ -82,22 +82,74 @@ If you want to add SMP Service to advertising data, refer to the :ref:`zephyr:sm
 
 .. fota_upgrades_over_ble_additional_information_end
 
+.. fota_upgrades_over_ble_mcuboot_direct_xip_information_start
+
+Build configuration additions for MCUboot in the direct-xip mode
+----------------------------------------------------------------
+
+FOTA updates are also supported when MCUboot is in the direct-xip mode.
+In this mode, the MCUboot bootloader boots an image directly from a given slot, so the swap operation is not needed.
+It can be used either with or without the revert mechanism support.
+For more information about the direct-xip mode and the revert mechanism support, go to the Equal slots (direct-xip) section on the :doc:`mcuboot:design` page.
+
+To use MCUboot in the direct-xip mode together with FOTA updates, do the following:
+
+* Enable the :kconfig:option:`CONFIG_BOOT_DIRECT_XIP` Kconfig option in the MCUboot bootloader configuration.
+* Enable the :kconfig:option:`CONFIG_MCUBOOT_BOOTLOADER_MODE_DIRECT_XIP` Kconfig option to inform the application about the chosen MCUboot mode of operation.
+
+   * This Kconfig option automatically enables the :kconfig:option:`CONFIG_BOOT_BUILD_DIRECT_XIP_VARIANT` Kconfig option to build application update images for both slots.
```

Comment:
This part is the same in the case of the direct-xip mode with revert and without revert. Can we have a general comment after each mode description to avoid duplication? Any ideas @annwoj?

Reply:
Yeah, I wanted to include this note however i had no idea how to avoid writing it explicite.

Reply:
Replaced with one general note after everything, please take a look if it is acceptable for you.